src/Viewer/services/decoder/FourByteClpIrStreamReader.js

```diff
@@ -211,6 +213,20 @@ class FourByteClpIrStreamReader {
         return FourByteClpIrStreamReader.VERBOSITIES.length - 1;
     }
 
+    /**
+     * We store the log templates in `typeIndexDict` and assign each of them a unique index.
+     * This functions return the corresponding index of the current `this._logtype`.
+     */
+    _getTypeIndex () {
+        const s = FourByteClpIrStreamReader.textDecoder.decode(this._logtype._valueUint8Array);
+        const d = FourByteClpIrStreamReader.typeIndexDict;
```

Comment:
I know `d` here stands for a dictionary of type indices. Can we elaborate on what `s` represent here? (Speaking from a person who has no CLP background).